apps/pie-docs/src/_11ty/shortcodes/card.js

```diff
@@ -1,45 +1,99 @@
 const pieIconsSvg = require('../filters/pieIconsSvg');
+const pieDesignTokenColours = require('../filters/pieDesignTokenColours');
 
 /**
  * A Card HTML component – takes an array of list items and turns them into a list of cards
  * @param {string[]} items - An array of card items
```

Comment:
is this correct? I thought it was an array of item objects. in which case we should try and define their properties here too :)